### ISSUE TITLE
State of CC0 waiver next button unconditionally retained

### ISSUE THREAD

**Issue by fkohrt** [creativecommons/chooser#323]
## Description

For an edge case at the CC0 step _Waive Your Copyright_, the checkboxes' states are incongruent with the `disabled` state of the _Next_ button.

## Reproduction

There are two paths to the CC0 license:

- Choosing it directly via _I know the license I need._
- Opting out of attribution via _Anyone can use my work, even without giving me attribution._

Going down one path until _Waive Your Copyright_, ticking both options and then going back and choosing the other path will result in the described incongruency.

## Expectation

_Next_ button should be disabled when arriving at _Waive Your Copyright_ (or checkboxes should save their state as well).

## Environment

 - Device: Laptop
 - OS: Fedora 32
 - Browser: Firefox
 - Version: 85.0.1

## Resolution

- [ ] I would be interested in resolving this bug.

**Comment by neeraj-2** [comment]
I would like to work on it. can I ?

**Comment by fkohrt** [comment]
I'm not part of the CC community team, but sure, fine by me.

**Comment by fkohrt** [comment]
But note that the issue is still awaiting triage and therefore generally not considered ready for work (see the [Contribution Guidelines](https://opensource.creativecommons.org/contributing-code/)).

**Comment by neeraj-2** [comment]
I think we can have two approaches :
1. Make the Next button disabled again and in this way, user can tick both choices again and thus we can make Next button in enables state. or,
2. Remember the state chosen by the user and this making the Next button in the enabled state.
                        I would prefer the second approach in which we can remember the state chosen by the user. Want your suggestions @zackkrida  and @obulat

**Comment by neeraj-2** [comment]
> But note that the issue is still awaiting triage and therefore generally not considered ready for work (see the [Contribution Guidelines](https://opensource.creativecommons.org/contributing-code/)).

Yeah @fkohrt, I would start working only after it would be assigned or tagged as ready for work. Thanks.

**Comment by zackkrida** [comment]
The bug here is that the 'Waive Your Copyright' checkboxes should _stay checked_ when you go back to the step. Any can start working on that fix!

**Comment by neeraj-2** [comment]
Thanks, @zackkrida will create a PR soon.
